### PR TITLE
test: add account linking e2e tests

### PR DESCRIPTION
**Pull request by kleyow**

### CODE REVIEW

**Comment by kleyow** [comment]
Andddd here are e2e tests. All that is left is waiting for the `thirdparty-scheme-adapter` https://github.com/mojaloop/thirdparty-scheme-adapter/pull/102 PR to pass and be built and a simple swap in the compose file which is building off the PR. 

Once the auth-service is ready it should be a simple swap of images and updating the signed credential dummy data.

**Comment by lewisdaly** [comment]
@kleyow is this ready for review? Or do you want to wait for the 3p-scheme-adapter stuff as you mentioned above?

**Comment by kleyow** [comment]
@lewisdaly pushing the compose files changes right now. you can review it now

File: docker-local/postman/scripts/_00_seed_all.sh [hunk 1]
```diff
@@ -14,6 +14,7 @@ set -u
 ./_03_seed_dfspa.sh
```
Comment (lewisdaly) [inline docker-local/postman/scripts/_00_seed_all.sh:14]: FYI - I'm currently working on replacing this tower of cards with `ml-bootstrap`

**Review by lewisdaly — APPROVED** [review]

File: docker-local/postman/scripts/_00_seed_all.sh (hunk 1 above)
Comment (kleyow) [reply docker-local/postman/scripts/_00_seed_all.sh:14]: 🙏

**Pull request merged**